components/require.js

```diff
@@ -0,0 +1,2044 @@
+/** vim: et:ts=4:sw=4:sts=4
+ * @license RequireJS 2.1.5 Copyright (c) 2010-2012, The Dojo Foundation All Rights Reserved.
```

Comment:
Seems like this whole Require.js stuff could be referenced instead of copy/pasted into the project. Why not just link to this: http://requirejs.org/docs/release/2.2.0/minified/require.js ?